src/bmffimage.cpp

```diff
@@ -107,7 +107,10 @@ namespace Exiv2
         std::string result;
         bool bBigEndian = isBigEndianPlatform();
         for (int i = 0; i < 4; i++) {
-            result += p[bBigEndian ? i : (3 - i)];
+            char c = p[bBigEndian ? i : (3 - i)];
+            result += (32<=c && c<=127) ? c      // only allow 7-bit printable ascii
```

Comment:
Off by one. ASCII character 127 is DEL and not printable.
```
c<127

Reply:
Fixed.